versioned_docs/version-24.04/releases/centreon-os.mdx

```diff
@@ -193,6 +193,24 @@ Many features have been added in this version.
 
 ## Centreon Collect
 
+### 24.04.3
+
+Release date: `July 17, 2024`
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [Broker] Fixed an issue while inserting metric into the database, which resulted with error log but Broker continues to work.
+- [Broker] If a metric can't be inserted in the database, the metric is dropped with an error log but Broker continues to work, and others metrics are still inserted.
+- [Broker] There was two errors concerning the gnutls version check:
+1. We wanted that the gnutls library on the computer is at least the same as the one used to compile broker.
+2. The packaging required a very old version of gnutls 3.3.29
+
+Now, we expect that the version of the gnutls library is at least 3.6.0. And in the packaging, broker will be installed only if a such gnutls library can be installed.
+- [Broker] Fixed an issue while trying to Connect to MySQL 8.0.35.
```

Comment:
```suggestion
- [Broker] Fixed an issue that occurred when inserting too large metrics into the database, which resulted in an error log.
- [Broker] There were two errors concerning the gnutls version check:
1. The gnutls library on Centreon had to be at least the same version as the one used to compile Broker.
2. The packaging required a very old version of gnutls.
Now, Broker checks that the version of the gnutls library is at least 3.6.0. In the packaging, when Broker is installed we check that the version of gnutls is at least 3.6.0. If the version is older, it will be updated.
- [Broker] Added support for the default connection method to MySQL 8.0.35.
```